integrations/test_trec_covid_r3.py

```diff
@@ -0,0 +1,81 @@
+#
+# Pyserini: Python interface to the Anserini IR toolkit built on Lucene
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import hashlib
+import os
+import re
+import shutil
+import unittest
+import json
+
+import sys
+sys.path.append('..')
+
+from random import randint
+from pyserini.util import download_url,download_prebuilt_index
+from integrations.simplesearcher_checker import SimpleSearcherChecker
+
+
+
+class TestSearchIntegration(unittest.TestCase):
+    def setUp(self):
+        self.round4_runs = {
+            'https://raw.githubusercontent.com/castorini/anserini/master/src/main/resources/topics-and-qrels/qrels.covid-round3-cumulative.txt':
+                'dfccc32efd58a8284ae411e5c6b27ce9',
+            'https://raw.githubusercontent.com/castorini/anserini/master/src/main/resources/topics-and-qrels/qrels.covid-round4-cumulative.txt':
+                '7a5c27e8e052c49ff72d557051825973'
+        }
+        self.tmp = f'tmp{randint(0, 10000)}'
+
+        # In the rare event there's a collision
+        if os.path.exists(self.tmp):
+            shutil.rmtree(self.tmp)
+
+        os.mkdir(self.tmp)
+        for url in self.round4_runs:
+            print(f'Verifying stored run at {url}...')
+            filename = url.split('/')[-1]
+            filename = re.sub('\\?dl=1$', '', filename)  # Remove the Dropbox 'force download' parameter
+
+            download_url(url, self.tmp, md5=self.round4_runs[url], force=True)
+            self.assertTrue(os.path.exists(os.path.join(self.tmp, filename)))
+            print('')
+
+
+    def test_bm25(self):
+
+        prebuilt_index_path = download_prebuilt_index('trec-covid-r4-abstract')
+        os.system(f'python ../trec-covid-r3/ranker.py \
```

Comment:
This file doesn't exist, so I can't run the integration test...
Should probably be in `scripts/`

Reply:
The ranker.py should be under trec-covid-r3. Can I know where did you put the trec-covid-r3?

Reply:
Also, to run the script, we should keep a ../trec-covid-r3/data/covidex.r4.d2q.duot5 and ../trec-covid-r3/data/covidex.r4.d2q.duot5.lr files. Should I assume the user contain these two files or download them in the setup?

Reply:
You can download the runs to `runs/`

Reply:
Done it. Also pls pull the newest trec-covid-r3 for testing. Thanks a lot!